### DISCUSSION TITLE
Install script?

### DISCUSSION THREAD

**Discussion by MorphyDK** [Novik/ruTorrent#2738] [Ideas]
Hi :)

Just curious why you don't make an install script? 

Tbh I find rutorrent the best client and have been very happy using https://github.com/arakasi72/rtinst . 

Now my server crashed today after several years of running :) - and I want to use his script again. I simply can't get it to work anymore due some php install issues. 

Then I saw this yesterday and want to give it a go, but then im lost. I admit im a total idiot when it comes to manual install of rutorrent. I simply don't know where to start .

I don't want a docker solution, I like my servers to run in proxmox and since my main docker server isn't that powerfull 

Hope that you might consider it :)

Thank you

**Comment by n1ce-0ne** [comment]
the script does still work just needs a few tweaks to run on the newer operating systems.

try https://github.com/n1ce-0ne/rtinst
